<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationHistory</class>
 <widget class="QWidget" name="NavigationHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recent</string>
  </property>
  <property name="windowIcon">
   <iconset>icons:history.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1" columnstretch="0,1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QPushButton" name="buttonJump">
     <property name="text">
      <string>&amp;Jumpâ€¦</string>
     </property>
     <property name="icon">
      <iconset>icons:jump_to_obs.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>814</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="buttonClear">
     <property name="toolTip">
      <string>Clear list of recentliy viewed observations</string>
     </property>
     <property name="text">
      <string>For&amp;get</string>
     </property>
     <property name="icon">
      <iconset>icons:forget.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableView" name="tableHistory">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
